pkg/image/validation/kubernetes_test.go

```diff
@@ -14,7 +18,47 @@ var validNetwork = image.Network{
 	APIVIP:  "127.0.0.1",
 }
 
+func setupContext(t *testing.T) (ctx *image.Context, teardown func()) {
+	configDir, err := os.MkdirTemp("", "eib-config-")
+	require.NoError(t, err)
+
+	buildDir, err := os.MkdirTemp("", "eib-build-")
+	require.NoError(t, err)
+
+	combustionDir, err := os.MkdirTemp("", "eib-combustion-")
+	require.NoError(t, err)
+
+	artefactsDir, err := os.MkdirTemp("", "eib-artefacts-")
+	require.NoError(t, err)
+
+	ctx = &image.Context{
+		ImageConfigDir:  configDir,
+		BuildDir:        buildDir,
+		CombustionDir:   combustionDir,
+		ArtefactsDir:    artefactsDir,
+		ImageDefinition: &image.Definition{},
+	}
+
+	return ctx, func() {
+		assert.NoError(t, os.RemoveAll(combustionDir))
+		assert.NoError(t, os.RemoveAll(buildDir))
+		assert.NoError(t, os.RemoveAll(artefactsDir))
+		assert.NoError(t, os.RemoveAll(configDir))
+	}
+}
+
 func TestValidateKubernetes(t *testing.T) {
+	ctx, teardown := setupContext(t)
+	defer teardown()
+
+	valuesDir := filepath.Join(ctx.ImageConfigDir, "kubernetes", "helm", "values")
+	err := os.MkdirAll(valuesDir, os.ModePerm)
+	require.NoError(t, err)
+
+	apacheValuesPath := filepath.Join(valuesDir, "apache-values.yaml")
+	err = os.WriteFile(apacheValuesPath, []byte(""), 0o600)
+	require.NoError(t, err)
+
```

Comment:
What's the point of all of this?

Reply:
This is the only real way to test the Helm values file logic when it's working and in a non-error case